cdap-watchdog/src/main/java/co/cask/cdap/logging/context/LoggingContextHelper.java

```diff
@@ -127,19 +127,31 @@ public static LoggingContext getLoggingContext(String namespaceId, String applic
 
   public static LoggingContext getLoggingContext(String namespaceId, String applicationId, String entityId,
                                                  ProgramType programType, @Nullable String adapterName) {
+    return getLoggingContext(namespaceId, applicationId, entityId, programType, null, adapterName);
+  }
+
+  public static LoggingContext getLoggingContextWithRunId(String namespaceId, String applicationId, String entityId,
```

Comment:
Is this method necessary? It only enforces a runId (via the absence of ``@Nullable``, right? can we directly call the method below?